settings.py

```diff
@@ -92,9 +92,9 @@
     'compressor.finders.CompressorFinder',
 )
 
-STATICFILES_DIRS = (
+STATICFILES_DIRS = [
```

Comment:
Curious if/why this needed to change.

Reply:
It was telling me that tuple addition wasn't allowed under certain circumstances when I was adding formdesigner a couple lines below

Also it seemed like a list was semantically more appropriate than a tuple anyway

And finally, [in the docs](https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-STATICFILES_DIRS) it's set as a list, so it didn't seem like it would break too much